modules/processor/header-enricher/config/header-enricher.groovy

```diff
@@ -26,7 +27,7 @@ beans {
 	
 	si.'header-enricher'('input-channel':'input','output-channel':'output') {
 		result.each {k,v->
-			si.'header'(name:k,expression:v)
+			si.'header'(name:k,expression:v,overwrite:overwrite)
```

Comment:
Of course, the next request we'll get is how to overwrite some but not others - but this is an important good start; they can chain two HEs as a work-around.